api/node/package.json

```diff
@@ -1,25 +1,49 @@
 {
-    "name": "slint-ui",
-    "version": "1.3.0",
-    "homepage": "https://github.com/slint-ui/slint",
-    "license": "SEE LICENSE IN LICENSE.md",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/slint-ui/slint"
-    },
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
-    "dependencies": {
-        "@types/node": "^14.11.11",
-        "neon-cli": "^0.4",
-        "typescript": "^4.0.3"
-    },
-    "scripts": {
-        "install": "neon build --release && tsc",
-        "build": "tsc",
-        "docs": "typedoc --hideGenerator --readme cover.md lib/index.ts"
-    },
-    "devDependencies": {
-        "typedoc": "^0.19.2"
+  "name": "slint-ui",
+  "version": "1.3.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "license": "GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial",
```

Comment:
Reading https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license I think we need to use the old variant of this field. What do you think?

Reply:
@tronical do you mean with the `SEE LICNESE ... LICENSE.md`?

Reply:
Yes, this section "If you are using a license that hasn't been assigned an SPDX identifier, or if you are using a custom license, use a string value like this one".

Reply:
ok that's done.